solver/src/driver.rs

```diff
@@ -263,8 +263,13 @@ impl Driver {
         &self,
         settlements: Vec<SettlementWithSolver>,
         prices: &HashMap<H160, BigRational>,
+        gas_price_wei: f64,
     ) -> Vec<RatedSettlement> {
         use futures::stream::StreamExt;
+
+        // TODO: 1e18 should be 10**(native_token_decimals)
+        let gas_price_wei = BigRational::from_float(gas_price_wei).unwrap();
```

Comment:
Is there any way that this can fail? In my understanding of rust lang, we should avoid unwraps everywhere but tests or at least comment here on why we know that this will never result in an Error.